src/ComposeUI.Example.WPFDataGrid/ShellView.xaml.cs

```diff
@@ -0,0 +1,157 @@
+﻿// /*
+//  * Morgan Stanley makes this available to you under the Apache License,
+//  * Version 2.0 (the "License"). You may obtain a copy of the License at
+//  *
+//  *      http://www.apache.org/licenses/LICENSE-2.0.
+//  *
+//  * See the NOTICE file distributed with this work for additional information
+//  * regarding copyright ownership. Unless required by applicable law or agreed
+//  * to in writing, software distributed under the License is distributed on an
+//  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+//  * or implied. See the License for the specific language governing permissions
+//  * and limitations under the License.
+//  */
+
+using ComposeUI.Example.WPFDataGrid.Models;
+using ComposeUI.Messaging.Client;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Windows;
+using System.Windows.Controls;
+
+
+namespace ComposeUI.Example.WPFDataGrid;
+
+/// <summary>
+/// Interaction logic for ShellView.xaml
+/// </summary>
+public partial class ShellView : Window
+{
+    private readonly ILogger<ShellView>? _logger;
+    private readonly IMessageRouter _messageRouterClient;
+    private readonly List<SymbolModel> _symbols;
+
+    /// <summary>
+    /// Constructor for the View.
+    /// </summary>
+    /// <param name="logger"></param>
+    /// <param name="messageRouterClient"></param>
+    public ShellView(ILogger<ShellView> logger, IMessageRouter messageRouterClient)
```

Comment:
Minor nitpicking on naming: it could be just `messageRouter` and `_messageRouter`. When naming variables, the general rule of thumb is you use the type name unless it's misleading or not descriptive enough.